mock-mhs-adaptor/src/main/resources/application.yml

```diff
@@ -1,6 +1,13 @@
 server:
   port: ${MOCK_MHS_SERVER_PORT:8081}
 
+logging:
+  level:
+    uk:
+      nhs:
+        adaptors:
+          mockmhsservice: DEBUG
```

Comment:
Minor,  but I think you can just do

```
logging:
  level:
    root: DEBUG
```

Reply:
👍  took the second option, we're not interested in debug for third party stuff